coordinator/Dockerfile

```diff
@@ -1,20 +1,30 @@
-FROM openjdk:22-ea-17-slim-bookworm
+FROM openjdk:21-jdk-slim AS builder
 
 RUN apt-get update \
-	&& apt-get install curl -y \
   && apt-get install unzip -y \
 	&& apt-get clean \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN mkdir -p src libs
```

Comment:
This would create dir under the root `/` ?

Reply:
Wherever WORKDIR is defined